src/lib/viewers/BaseViewer.js

```diff
@@ -45,6 +45,8 @@ const VIEWER_STATUSES = {
     loading: 'loading',
 };
 
+export const CSS_ANNOTATIONS_DISCOVERABLE_CLASS = 'bp-annotations-discoverable';
```

Comment:
Maybe we can put this with the other constants in the `constants.js`? And also the naming convention for those constants seems to be `CLASS_*`